doc/install/clone-source.rst

```diff
@@ -81,9 +81,85 @@ repository.
 	cd ceph
 	git status
 
-If your submodules are out of date, run::
-
-	git submodule update --force --init --recursive
+.. _update-submodules:
+
+Updating Submodules
+-------------------
+
+#. Determine whether your submodules are out of date:
+
+   .. prompt:: bash $
+
+      git status
+
+   A. If your submodules are up to date 
```

Comment:
I often see up-to-date but you're the expert here ;)

Reply:
If it's an adjective before a noun, it has hyphens. Here it's a predicate adjective, so: no hyphens.

That's English. Thank God we're native speakers.